plugins/basic/nexus-repository-nuget/src/main/java/com/sonatype/nexus/repository/nuget/internal/NugetGalleryFacetImpl.java

```diff
@@ -604,6 +607,15 @@ void setDerivedAttributes(final Map<String, String> incomingMetadata,
         (nullToEmpty(incomingMetadata.get(TITLE)) + nullToEmpty(incomingMetadata.get(ID))).toLowerCase());
   }
 
+  private int parseIntOrZero(final String s) {
+    try {
+      return Integer.parseInt(s);
+    }
+    catch (NumberFormatException e) {
```

Comment:
I don't think so - most of the time the problem is that it's blank rather than something entirely non-numeric.